docs/source/tracking.rst

```diff
@@ -750,6 +833,18 @@ location to server's artifact store. This will be used as artifact location for
 experiments that do not specify one. Once you create an experiment, ``--default-artifact-root``
 is no longer relevant to that experiment.
 
+Starting a server with the ``--serve-artifacts`` option defined will enable proxy access for artifacts.
+The uri ``mlflow-artifacts:/`` will serve in place of the ``--default-artifact-root`` path configured during
+server start to refer to artifact storage locations.
```

Comment:
```suggestion
The URI ``mlflow-artifacts:/`` is the default value for ``--default-artifact-root`` when `--serve-artifacts` is specified. This indicates to clients that artifacts can be accessed via HTTP requests to the MLflow Tracking Server.
```

Reply:
Can we also enumerate and explain the use cases for some other proxied artifact `--default-artifact-root` values? E.g. `mlflow-artifacts://<tracking_server_host_name>/sub/path` or `https://<host_name>/sub/path`?

Reply:
Added a few listed examples - please check to make sure I'm capturing what you're looking for here or if I completely missed the plot.